docs/src/components/codes/OsSupportIcon.jsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: CC-BY-NC-SA-4.0
+ */
+
+import React from "react";
+import PropTypes from "prop-types";
+
+const Icon = ({ children, className, title }) => (
+  <span className={className} title={title}>
+    {children}
+  </span>
+);
+
+Icon.propTypes = {
+  children: PropTypes.oneOfType([PropTypes.element, PropTypes.string])
+    .isRequired,
+  className: PropTypes.string.isRequired,
+  title: PropTypes.string.isRequired,
+};
+
+export const Supported = () => (
+  <Icon className="supported" title="Supported 😄">
+    ★
+  </Icon>
+);
+export const NotSupported = () => (
+  <Icon className="not-supported" title="Not Supported 😢">
+    🞨
+  </Icon>
+);
+export const NotTested = () => (
+  <Icon className="not-tested" title="Not Tested Yet - PR's welcomed! 🧐">
+    ?
```

Comment:
These icons work for ok, but I kind of like the aesthetic of the emojis used on our feature table. Thoughts?

Reply:
Hehe, I actually thought the same ... but about the tick in the features table.  I discussed this on Discord a while back because I'd like a solution that's compatible across different cultures.  The affirmative meaning of the check mark is a very English concept.  It even has the opposite meaning in some cultures.  See https://en.wikipedia.org/wiki/Check_mark for more detail.

I felt the stars were clear enough to be understood without a legend and not too over-stated or under-stated (on the various OS renderings) to unbalance the table.  The 5-star ratings also gave me a giggle.  I also felt that the small cross was more useful than a empty space which could be misinterpreted.  The question mark is pretty self explanatory?

Taking the above into account, any thoughts?

I made sure they all have titles/mouseover hints.

Reply:
I think I've found some values I like:

- Windows: #CAEDFD
- Linux: #FFF2CA
- Android: #D8EED9

And then I'm thinking iOS could just be white and MacOS could take iOS's current color.

Result:
![image](https://user-images.githubusercontent.com/9439650/97750694-4bef9400-1abf-11eb-9bd1-29260aeadc2b.png)

We also may be able to add `mix-blend-mode: multiply` to `.os`. It keeps the every other row difference in brightness, but it looks terrible in dark mode. If we could turn it off in dark mode, it would be an improvement:
![image](https://user-images.githubusercontent.com/9439650/97751289-0d0e0e00-1ac0-11eb-9794-a1de538b6df8.png)

Reply:
Good points on all of this. Any reason we couldn't use the star, x, and ? emojis? I think they look a bit more lively and in-line with the site's style. Not super important to change that, just a thought!

Reply:
Could.  Which ones did you have in mind?  Emojis seemed quite distracting when I tried some previously.

Reply:
- Star: ⭐
- Cross: ❌
- Question mark: ❔

Maybe they'd be a bit too much, I can see that.

Reply:
Gave it a go!  What do you think?